tests/python/unittest/test_gluon.py

```diff
@@ -135,6 +135,17 @@ def test_parameter_dict():
     mx.test_utils.assert_almost_equal(prev_w0.asnumpy(), cur_w0.asnumpy())
     mx.test_utils.assert_almost_equal(prev_w1.asnumpy(), cur_w1.asnumpy())
 
+    # test reset_ctx
```

Comment:
inaccurate comment

Reply:
Added `assert` relying on the previously tested `Parameter.list_ctx`